JumpRoyale/src/Arena.cs

```diff
@@ -140,7 +150,13 @@ private void HandleCommands(string message, string senderId, string senderName,
     private void SetBackground()
     {
         Sprite2D background = GetNode<Sprite2D>("Background");
-        string[] colors = ["Blue", "Brown", "Gray", "Green", "Pink", "Purple", "Yellow"];
+
+        // Note: old backgrounds, which still are present in the project
+        // Note2: the old backgrounds use "modulate = Color(1, 1, 1, 0.25098)" in the editor, because they are bright,
+        // the modulate has been removed for new textures and were made darker through image editing. Revert this change
+        // if the old backgrounds are used again.
+        // string[] colors = ["Blue", "Brown", "Gray", "Green", "Pink", "Purple", "Yellow"];
```

Comment:
Nit: delete these comments. The Git history can act as the reference point for their removal.

Reply:
Removed 👍

Reply:
I forgot to mention that we should delete the now-unused background graphics (e.g. `Blue.png`). We can do that in a later change at this point.